pkg/interfaces/contracts/liquidity-mining/ILiquidityGauge.sol

```diff
@@ -17,8 +17,12 @@ pragma solidity ^0.7.0;
 // For compatibility, we're keeping the same function names as in the original Curve code, including the mixed-case
 // naming convention.
 // solhint-disable func-name-mixedcase
+// solhint-disable func-param-name-mixedcase
 
 interface ILiquidityGauge {
+    // solhint-disable-next-line var-name-mixedcase
+    event MaxRelativeWeightChanged(uint256 indexed new_max_relative_weight);
```

Comment:
```suggestion
    event MaxRelativeWeightChanged(uint256 new_max_relative_weight);
```

There's no point in indexing this

Reply:
Thanks; I was hesitant about it too (maybe cap values are somewhat standard?).

Removed.